addons/event_sale/models/event_ticket.py

```diff
@@ -43,6 +43,10 @@ def _compute_description(self):
             if not ticket.description:
                 ticket.description = False
 
+    # FIXME WTF is this broll ?
```

Comment:
We call it "broken by design". I think it comes from first implementation of website_event, never fixed as pricelists were a PITA to handle. We could probably work on this after your merge

Short version: would update offensive comment with something like "now unblocked, can be cleaned in master"

Reply:
Pricelist mess is the next target (https://github.com/odoo/odoo/pull/79605 being the first step).
Wanted to highlight it to know what should be done for this broken binz until then, but indeed, comment should be pacified before merge :D

Reply:
Basically we know this has to be cleaned and we were waiting for sale to finally be compute-ready. Cleaning usage of pricelists and real product price was about impossible as code cannot be called cleanly. So yep, would pacify the comment ^^ .